src/XenCert/XenCertCommon.py

```diff
@@ -132,8 +133,8 @@ def store_configuration(g_storage_conf, options):
 
 def valid_arguments(options, g_storage_conf):
```

Comment:
lvmohba and lvmoiscsi are kept here, whether the purpose is just making XenRT test without changes?

Reply:
I didn't want to remove the old values, people may be used to using them. I have separate XenRT changes which will enable the tests to be run in XenRT.

Reply:
ah, I see what you mean now. Yes, this would remove some of the duplicated code

Reply:
Thanks!